templates/groupList.html

```diff
@@ -1 +1 @@
-<li ng-repeat="group in ctrl.groups" group data="group" ng-click="ctrl.setSelected(group)" ng-class="{active: group === ctrl.getSelected}"></li>
+<li ng-repeat="group in ctrl.groups" group group-name="group[0]" group-count="group[1]" ng-click="ctrl.setSelected(group[0])" ng-class="{active: group[0] === ctrl.getSelected}"></li>
```

Comment:
The order of keys in an object is arbitrary, so I don't want to have the whole `ctrl.groups` be an object. We want to guarantee `All Contacts` is first.